cmd.go

```diff
@@ -504,3 +565,52 @@ func CreateMoonstreamCommand() *cobra.Command {
 
 	return moonstreamCommand
 }
+
+func CreateServerCommand() *cobra.Command {
```

Comment:
Maybe better to have:
- `waggle server configure` (replacing `waggle accounts config`)
- `waggle server run`